scripts/vast-completions.bash

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+_vast_completions()
+{
+  completion_words=("${COMP_WORDS[@]}")
+  if [ "${COMP_WORDS[@]: -1}" == "--" ]; then
+    # We need to special-case '--', since that has special
+    # meaning, e.g.
+    #
+    #    vast pivot --asdf help -> displays error message/help text
+    #    vast pivot -- help     -> runs pivot command
+    #
+    # Technically we should scan the whole COMP_WORDS array
+    # for '--', but that issue should come up rarely, wheras
+    # a '--' at the end happens all the time as the result of
+    # a previous autocompletion.
+    completion_words[$((${#COMP_WORDS}-1))]="-"
+  fi
+  tmpfile=$(mktemp)
+  ${completion_words[@]} help 2>&1 |  awk '/subcommands:/{x = 1} !/subcommands:/{if (x > 0) print $1} match($0, /--[^\]]*]/) { print substr($0, RSTART, RLENGTH-1) }' >${tmpfile}
+  readarray -t commands < ${tmpfile}
```

Comment:
```suggestion
  readarray -t commands < ${tmpfile}
```

This is the only really non-portable line between zsh and bash in the whole script (assuming `autoload bashcompinit`). Can we get rid of `readarray` somehow?